src/components/header.js

```diff
@@ -21,26 +22,29 @@ function Header() {
     if (path !== '/')
```

Comment:
I'm going to have to run a global search on path and see if '/' has to do with the root directory.

Reply:
This is something that John introduced but, as I understand it, it checks during run-time whether the user is on the main page or whether they're on another page like About or Code of Conduct. It's related to the current page, not to the source file folders.